evm_loader/program/src/error.rs

```diff
@@ -26,4 +26,34 @@ impl<T> DecodeError<T> for EvmLoaderError {
     fn type_of() -> &'static str {
         "EVMLoaderError"
     }
-}
\ No newline at end of file
+}
+
+pub fn err_fn_without_info<T:>(err: ProgramError, fl: &str, ln: u32) -> Result<T, ProgramError> {
+    solana_program::msg!("{}:{:?}", fl, ln);
+    Err(err)
+}
+
+pub fn err_fn<T:>(err: ProgramError, fl: &str, ln: u32, info: &str) -> Result<T, ProgramError> {
+    solana_program::msg!("{}:{:?} : {}", fl, ln, info);
+    Err(err)
+}
+
+macro_rules! Err {
```

Comment:
Maybe it would make sense to add comments describing details and differencies of macros.

Reply:
Fixed